documentation/book/proc-manual-restart-zookeeper.adoc

```diff
@@ -0,0 +1,41 @@
+// Module included in the following assemblies:
+//
+// assembly-deployment-configuration-kafka.adoc
+
+[id='proc-manual-restart-zookeeper-{context}']
+= Manually restart a Zookeeper cluster
+
+This procedure describes how to manually restart an existing Zookeeper cluster by using an {ProductPlatformName} annotation.
+
+.Prerequisites
+
+* A running Zookeeper cluster.
+* A running Cluster Operator.
+
+.Procedure
+
+. Find the name of `StatefulSet` which controls Zookeeper pods to restart.
++
+Lets say the cluster is named _my-cluster_. The name of the Zookeeper StatefulSet is _my-cluster-zookeeper_.
++
+
+. Annotate `StatefulSet` resource in {ProductPlatformName}.
++
+ifdef::Kubernetes[]
+On {KubernetesName} this can be done using `kubectl annotate`:
+[source,shell,subs=+quotes]
+kubectl annotate statefulset _cluster-name_-zookeeper operator.strimzi.io/manual-rolling-update=true
+endif::Kubernetes[]
++
+On {OpenShiftName} this can be done using `oc annotate`:
+[source,shell,subs=+quotes]
+oc annotate statefulset _cluster-name_-zookeeper operator.strimzi.io/manual-rolling-update=true
++
+. Wait for next reconciliation and for all pods in annotated `StatefulSet` rolling update will be done.
```

Comment:
_Change this step as above:_

3. Wait for the next reconciliation to occur (every two minutes by default).

    All pods within the annotated ``StatefulSet`` are restarted on a rolling basis, as long as the annotation was detected by the reconciliation process.